labgrid/driver/smallubootdriver.py

```diff
@@ -58,6 +58,8 @@ def _await_prompt(self):
 
         # wait until UBoot has reached it's prompt
         self.console.expect(self.prompt)
+        for command in self.init_commands:  #pylint: disable=not-an-iterable
+            self._run(command)
```

Comment:
`self._run` is correct here instead of `self._run_check`, since the `SmallUBootDriver` returns `0` for every command anyway.

Reply:
Yep, it doesn't work with `_run_check`.